configure.ac

```diff
@@ -38,6 +38,13 @@ AC_ARG_ENABLE(
 	[enable_distonly="no"]
 )
 
+AC_ARG_ENABLE(
+	[password-save],
+	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
+	[enable_password_save="yes"],
+	[enable_password_save="no"]
+)
```

Comment:
This does not look right. The default is yes, so the last arg to AC_ARG_ENABLE would be enable_password_xxx ="yes".  And the third arg may be left blank.